src/openagi/worker.py

```diff
@@ -117,13 +185,24 @@ def execute_task(self, task: Task, context: Any = None) -> Any:
 
         logging.debug("Provoking initial thought observation...")
         initial_thought_provokes = self.provoke_thought_obs(None)
+
+        knowledge_base_info = "No relevant information found."
+        if self.knowledge_base:
+            query_results = self.knowledge_base.query_documents(
+                query_texts=[query],
+                n_results=3
```

Comment:
make it configurable. Have a default value so that it can be updated based on the requirement.

Reply:
@shreehari-aiplanet sure

Reply:
Done